Cogs/ClassManagement.py

```diff
@@ -0,0 +1,256 @@
+            return
+        
+        permissions = discord.Permissions(read_messages=True, send_messages=True, embed_links=True, 
+                attach_files=True, read_message_history=True, add_reactions=True, connect=True, speak=True, 
+                stream=True, use_voice_activation=True, change_nickname=True, mention_everyone=False)
+        
+        for i in range(len(role_names)):
+            # Create roles
+            role_name = role_names[i]
+            role = await ctx.guild.create_role(name=role_name, permissions=permissions)
+            role.mentionable = True
+            
+            # Create category
+            category_name = categories[i]
+            category = await ctx.guild.create_category(category_name)
+            await category.set_permissions(ctx.guild.default_role, read_messages=False)      # sets category to private
+            await category.set_permissions(role, read_messages=True)        # allow role to see category
+            
+            # Create channels
+            channels = class_channels[i].split(",")
+
+            for channel in channels:
+                # Create text channel
+                if channel.startswith('#'):
+                    await category.create_text_channel(channel, topic=long_names[i])
+                # Create voice channel(s)
+                else:
+                    member_count, channel_name = channel.split('#')
+                    if member_count == 0:
+                        await category.create_voice_channel(channel_name)
+                    else:
+                        await category.create_voice_channel(channel_name, user_limit=int(member_count))
+        await ctx.send('***CATEGORIES AND ROLES HAVE BEEN BUILT***')
+
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def destroycourses(self, ctx):
+        """Destroy course channels
+
+        Try loading in cached classlist csv
+        Create list of all channels to be deleted, send to author
+        Get confirmation from author
+        Delete all listed categories and channels
+        Create list of all roles to be deleted, send to author
+        Get confirmation from author
+        Delete all listed roles
+        """
+
+        # Load roles csv
+        csv_filepath = f'role_lists/roles_{ctx.guild.id}.csv'
+
+        # If the role_lists directory doesn't exist raise a FileNotFoundError back to the buildserver command
+        try:
+            courses_df = pd.read_csv(csv_filepath)
+        except FileNotFoundError:
+            raise FileNotFoundError
+
+        # drop cross-listed courses (and other roles if on the file)
+        courses_df = courses_df.dropna(subset=['create_channels'])
+
+        # List of names of categories to be destroyed, as determined by saved csv
+        destroy_category_names = courses_df['text'].tolist()
+        destroy_categories = await self.get_category(ctx, destroy_category_names)
```

Comment:
It might be better to error out if the categories and roles can't be found.

Right now there's some weirdness when using build and destroy.